plugins/user-settings/src/components/FeatureFlags/UserSettingsFeatureFlags.tsx

```diff
@@ -62,16 +63,25 @@ export const UserSettingsFeatureFlags = () => {
   return (
     <InfoCard title="Feature Flags">
       <List dense>
-        {featureFlags.map(featureFlag => {
+        <ListItem>
+          <TextField
```

Comment:
Can we set a threshold for this to appear (e.g. `featureFlags.length >= 10`)? I’m thinking it will be common to have only a few active feature flags.

Reply:
I'm not sure if I understood correctly! On our internal app, we have many `featureFlags`. It will be confusing for users if they can't see all their active feature flags!

Reply:
Oh definitely, all feature flags should be visible. I meant that the search box itself should only appear if enough feature flags exist to make search needed. If there's only 3 flags, for example, a search may add more clutter than actual utility. 😄

Reply:
aha, that makes sense 👍

Reply:
I added to show it if items are more than x, but it's working because when the number of items goes under the x, the filter is gone!